bootstrap_packer.py

```diff
@@ -0,0 +1,51 @@
+import botocore
+import boto3
+import jinja2
+import logging
+import os
+import subprocess
+import json
+import sys
+from botocore.exceptions import ClientError
+
+# Initialise logging
+logger = logging.getLogger(__name__)
+log_level = os.environ["LOG_LEVEL"] if "LOG_LEVEL" in os.environ else "ERROR"
+logger.setLevel(logging.getLevelName(log_level.upper()))
+logging.basicConfig(
+    stream=sys.stdout,
+    format="%(asctime)s %(levelname)s %(module)s "
+           "%(process)s[%(thread)s] %(message)s",
+)
+logger.info("Logging at {} level".format(log_level.upper()))
+
+
+def handler(event, context):
```

Comment:
this is in a Lambda format and should change
```suggestion
def handler(event):
```
also feel free to name it something other than `handler`

Reply:
not used anyway, ripped it out.